ts/IterableInboxMessageListItem.tsx

```diff
@@ -11,27 +11,50 @@ import Message from './messageType'
 
 type MessageListItemProps = {
    message: Message,
+   customization: {[key: string]: any}
 }
 
-const IterableInboxMessageListItem = ({ message }: MessageListItemProps) => {
+const IterableInboxMessageListItem = ({ message, customization }: MessageListItemProps) => {
    const messageTitle = message.inboxMetadata.title
    const messageBody = message.inboxMetadata.subtitle
    const messageCreatedAt = message.createdAt
 
+   function customizeStyles(styles: {[key: string]: any}, customization: {[key: string]: any}) {
```

Comment:
I don't like this implementation as it has a lot of moving parts that are somewhat confusing to understand and it also seems like a roundabout way of implementing customization, but we can fix that in the next ticket. I was thinking that the customization props file just has a bunch of named keys/variables that the implementer would just create, and the rest of the code would just copy from, rather than doing something like this.

Also, we should separate the styling of the last row in a list from this. They are semantically separate things going on. And theoretically we should be able to integrate that code with either the main inbox rendering code or the message list code.

Reply:
We can discuss this. My idea was that the customization would revert to a default styling if the customer does not provide a customization for that element. Putting it into one function that automates this seemed like a good idea.